fseai/fseai_models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Deep Learning models in fseai
+"""
+import torch.nn as nn
+
+
+class FCmodel(nn.Module):
+    def __init__(self, input_size, hidden_size, output_size):
+        super(FCmodel, self).__init__()
```

Comment:
python 2.7 syntax. In python 3, the preferred way is `super().__init__()`.
Although, you'll see a lot of people using this old way to keep it compatible with 2.7 and 3.x, I think we should stick to the latter